test/support/rtmp_source_test_pipeline.ex

```diff
@@ -0,0 +1,56 @@
+defmodule Membrane.RTMP.Source.TestPipeline do
+  @moduledoc false
+  use Membrane.Pipeline
+
+  import Membrane.ParentSpec
+
+  alias Membrane.RTMP.SourceBin
+  alias Membrane.Testing
+
+  @impl true
+  def handle_init(socket: socket, test_process: test_process) do
+    spec = %Membrane.ParentSpec{
+      children: [
+        src: %SourceBin{
+          socket: socket,
+          validator: Membrane.RTMP.Source.TestValidator
```

Comment:
I think that we should probably use `Verifier` instead of `Validator`, validation is usually about the data format which in our case is alright, we need to verify if we can let the stream to continue with given set of parameters.

Reply:
Also, can you create a test of what happens if the `Verifier` denies a particular stream? Also we can probably send a notification to the pipeline that a stream has been denied or something.